index.js

```diff
@@ -6,6 +6,8 @@
 
 const { existsSync, readFileSync } = require('fs')
 const { join } = require('path')
+const { createRequire } = require('node:module')
+require = createRequire(__filename)
```

Comment:
Why not const require make it local?

Reply:
🤷 the node docs recommend it like this, i assume theres some static analysis things at work here